core/arch/asm_defines.asm

```diff
@@ -910,27 +910,38 @@ ASSUME fs:_DATA @N@\
         mov      ARG1, p1   @N@\
         blx      callee
 #elif defined(RISCV64)
+/* For RISC-V, there is not a instruction could operate both immediate
```

Comment:
grammar: s/not a/no/; s/could/which can/; s/operate/operate on/; s/immediate/immediates/

Reply:
Done